caravel/assets/visualizations/nvd3_vis.js

```diff
@@ -295,8 +312,9 @@ function nvd3Vis(slice) {
         }
 
         if (fd.bottom_margin) {
-          chart.margin({ bottom: fd.bottom_margin });
+          chart.margin({ bottom: Math.max(fd.bottom_margin, stretchMargin) });
```

Comment:
I found that fd.bottom_margin is default set to be 50, so fd.bottom_margin is always true, if we only stretch the margin when fd.bottom_margin is false, then it will never happen?